test/ReactWrapper-spec.jsx

```diff
@@ -1807,6 +1807,12 @@ describeWithDOM('mount', () => {
       expect(children.at(1).hasClass('baz')).to.equal(true);
     });
 
+    it('should not attempt to get an instance for text nodes', () => {
+      const wrapper = mount(<div>B<span />C</div>);
+      const children = wrapper.children();
+      expect(children.length).to.equal(1);
```

Comment:
i agree it shouldn't attempt to get an instance - but the div has 3 children, not 1.